common/model-views.cpp

```diff
@@ -2166,89 +2205,267 @@ namespace rs2
+                    "Most device do not guaranty consequitive frames to have conseuquitive frame numbers\n"
+                    "But it is true most of the time" });
+
+                if (profile.as<rs2::video_stream_profile>())
+                {
+                    stream_details.push_back({ "Hardware Size",
+                        to_string() << original_size.x << " x " << original_size.y, "" });
 
-            ImGui::SameLine();
+                    stream_details.push_back({ "Display Size",
+                        to_string() << size.x << " x " << size.y, 
+                        "When Post-Processing is enabled, actual display size of the frame may differ from original capture size" });
+                }
+                stream_details.push_back({ "Pixel Format",
+                    to_string() << rs2_format_to_string(profile.format()), "" });
 
-            std::string res;
-            if (profile.as<rs2::video_stream_profile>())
-                res = to_string() << size.x << "x" << size.y << ",  ";
-            label = to_string() << res << truncate_string(rs2_format_to_string(profile.format()),9) << ", ";
-            ImGui::Text("%s", label.c_str());
-            if (ImGui::IsItemHovered())
-            {
-                ImGui::SetTooltip("%s","Stream Resolution, Format");
+                stream_details.push_back({ "Hardware FPS",
+                    to_string() << std::setprecision(2) << std::fixed << fps.get_fps(), 
+                    "Hardware FPS captures number of frames per second produced by the device.\n"
+                    "It is possible and likely that not all of these frames will make it to the application." });
+                
+                stream_details.push_back({ "Viewer FPS",
+                    to_string() << std::setprecision(2) << std::fixed << view_fps.get_fps(),
+                    "Viewer FPS captures how many frames the application manages to render.\n"
+                    "Frame drops can occur for variety of reasons." });
+
+                stream_details.push_back({ "", "", "" });
             }
 
-            ImGui::SameLine();
+            const std::string no_md = "no md";
 
-            label = to_string() << "FPS: " << std::setprecision(2) << std::setw(7) <<  std::fixed << fps.get_fps();
-            ImGui::Text("%s", label.c_str());
-            if (ImGui::IsItemHovered())
+            if (timestamp_domain == RS2_TIMESTAMP_DOMAIN_SYSTEM_TIME)
+            {
+                stream_details.push_back({ no_md, "", "" });
+            }
+
+            std::map<rs2_frame_metadata_value, std::string> descriptions = {
+                { RS2_FRAME_METADATA_FRAME_COUNTER                        , "A sequential index managed per-stream. Integer value" },
+                { RS2_FRAME_METADATA_FRAME_TIMESTAMP                      , "Timestamp set by device clock when data readout and transmit commence. usec" },
```

Comment:
Disclamer - the units are mostly usec, but there are exceptions (SR300 with 100nsec units)